test/tests/misc/signal_handler/tests

```diff
@@ -0,0 +1,16 @@
+[Tests]
+	issues = '#12722'
```

Comment:
Indenting is strange. Can you tighten these up to our normal format? Did you know we have an input file formatter in the Python framework?